common/src/main/java/com/sequenceiq/cloudbreak/rotation/secret/RotationExecutor.java

```diff
@@ -0,0 +1,8 @@
+package com.sequenceiq.cloudbreak.rotation.secret;
+
+public interface RotationExecutor {
+
+    void rotate(RotationContext rotationContext);
```

Comment:
would it make sense to add a rollback method here also?

Reply:
yes, i have not started to add rollback related part of the framework, can be done when we start to work on it